activerecord/lib/active_record/connection_adapters/abstract/transaction.rb

```diff
@@ -52,6 +52,7 @@ def initialize(connection, options)
         @state = TransactionState.new
         @records = []
         @joinable = options.fetch(:joinable, true)
+        @uncommittable = options[:uncommittable]
```

Comment:
`options.fetch(:uncommittable, false)` to give it a clear default value vs falling back to `nil`